src/main/proto/metadata.proto

```diff
@@ -22,3 +22,14 @@ message SnapshotMetadata {
   // Kafka offset when this snapshot was taken for that partition.
   int64 maxOffset = 6;
 }
+
+message SearchMetadata {
+  // Name of search metadata
+  string name = 1;
+
+  // snapshot name
+  string snapshotName = 2;
+
+  // url
+  string url = 3;
```

Comment:
is `url` always going to be `s3Path` ?

Reply:
It can be any URI. The first part of the URI is a scheme which contains prefix like s3, hdfs, file etc..

Reply:
got it!